server/neptune/workflows/internal/test/context.go

```diff
@@ -0,0 +1,48 @@
+package test
```

Comment:
I don't see this being used anywhere. Can we drop it, or was this more prep for the future?